_layouts/doc_page.html

```diff
@@ -52,7 +52,12 @@
 
     <script>
     $(function() {
-      $(".toc").load("/docs/{{ version }}/toc.html");
+      $toc = $(".toc");
+      $toc.load("/docs/{{ version }}/toc.html", function () {
+        $toc.find('a').each(function(idx, el) {
+          el.href = el.href.replace('replace-with-version', '{{ version }}');
```

Comment:
@gianm see #286 for what I meant. Haven't been able to test locally yet, due to jekyll "not working for me™ right now", but that's the idea.